setup.py

```diff
@@ -63,7 +71,7 @@
     extras_require={
         'development': TESTS_REQUIRES + DOCS_REQUIRES,
     },
-    packages=find_packages(),
+    packages=find_namespace_packages(),
```

Comment:
Yeah this had it's reason because I couldn't persuade python to package dir with static files (javascript, css, etc) in the python package (and I didn't want to add `__init__.py` everywhere).

I followed setuptools docs on this:
https://setuptools.pypa.io/en/latest/userguide/datafiles.html#subdirectory-for-data-files

which states _"...In this case, the recommended approach is to treat `data` as a namespace package..."_